smart-contract-sale/Makefile

```diff
@@ -34,4 +40,4 @@ optimize:
 	@docker run --rm -v $(CURDIR):/code \
 		--mount type=volume,source=dcc_bilateral_cache,target=/code/target \
 		--mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
-		cosmwasm/rust-optimizer:0.12.6
+		cosmwasm/rust-optimizer$(ARCH):0.12.6
```

Comment:
@mjaffa-figure should work with m1 if you add this to ats-smart-contract

Reply:
although i guess @brashidnia-figure is fixing it with a diff action

Reply:
tomato tomáto